### PR TITLE
All-In-One: PSAMA to use its own trust store

### PR DESCRIPTION
**Pull request by Gcolon021**
All-In-One: PSAMA to use its own trust store

### CODE REVIEW

File: initial-configuration/jenkins/jenkins-docker/jobs/Create Root Certs in TrustStore/config.xml [hunk 1]
```diff
@@ -22,6 +23,15 @@
...
         keytool -import -keystore /usr/local/docker-config/wildfly/application.truststore -storepass password -noprompt -trustcacerts -alias letsencryptauthority2 -file lets-encrypt-r3.der -storetype JKS
+
+        <!--Remove existing psama trust store so it doesn't break-->
+        rm -rf /usr/local/docker-config/psama/application.truststore
+
+        curl https://letsencrypt.org/certs/isrgrootx1.der -o isrgrootx1.der
+        curl https://letsencrypt.org/certs/lets-encrypt-r3.der -o lets-encrypt-r3.der
+
+        keytool -import -keystore /usr/local/docker-config/psama/application.truststore -storepass password -noprompt -trustcacerts -alias letsencryptauthority1 -file isrgrootx1.der -storetype JKS
+        keytool -import -keystore /usr/local/docker-config/psama/application.truststore -storepass password -noprompt -trustcacerts -alias letsencryptauthority2 -file lets-encrypt-r3.der -storetype JKS
```
Comment (Luke-Sikina) [inline initial-configuration/jenkins/jenkins-docker/jobs/Create Root Certs in TrustStore/config.xml:34]: How hard would it be to be super clever and pull the auth0 tenant from that env and add the corresponding cert to the trust store in the initial config pipeline?

File: initial-configuration/pass.tmp [hunk 2]
```diff
@@ -1 +1 @@
-30Ik2r_^Y9CppDHHvmtbGzNf
```
Comment (Luke-Sikina) [inline initial-configuration/pass.tmp:1]: delete this file, add to gitignore

File: start-picsure.sh [hunk 3]
```diff
@@ -36,6 +36,10 @@ if [ -f $DOCKER_CONFIG_DIR/wildfly/application.truststore ]; then
   export TRUSTSTORE_JAVA_OPTS="-Djavax.net.ssl.trustStore=/opt/jboss/wildfly/standalone/configuration/application.truststore -Djavax.net.ssl.trustStorePassword=password"
 fi
 
+if [ -f $DOCKER_CONFIG_DIR/psama/application.truststore ]; then
+    export PSAMA_TRUSTSTORE_VOLUME="-v $DOCKER_CONFIG_DIR/psama/application.truststore:/usr/local/tomcat/conf/application.truststore"
```
Comment (Luke-Sikina) [inline start-picsure.sh:40]: is there a scenario where the store doesn't exist where it's still valid for psama to start? Maybe we crash out of the start script

**Review by Luke-Sikina — APPROVED** [review]

File: initial-configuration/pass.tmp (hunk 2 above)
Comment (Luke-Sikina) [reply initial-configuration/pass.tmp:1]: Love how git gaurdian misses this, but flips out on innocuous .envs

File: initial-configuration/jenkins/jenkins-docker/jobs/Create Root Certs in TrustStore/config.xml (hunk 1 above)
Comment (Gcolon021) [reply initial-configuration/jenkins/jenkins-docker/jobs/Create Root Certs in TrustStore/config.xml:34]: We can discuss in OH

**Pull request merged**